TrackingPerformance/Condor/condorJobs_reco.py

```diff
@@ -109,29 +145,37 @@
         need_to_create_scripts = True
 
         # Create aida output Dir
-        output_dir_aida = os.path.join(output_dir, "aida_outputs"); os.makedirs(output_dir_aida, exist_ok=True)
+        output_dir_aida = output_dir / "aida_outputs"
+        output_dir_aida.mkdir(exist_ok=True)
 
         arguments = (
-            #f" --GeoSvc.detectors=/afs/cern.ch/work/g/gasadows/k4geo/FCCee/CLD/compact/{DetectorModelList_[0]}_3T/{DetectorModelList_[0]}.xml"+
-            f" --GeoSvc.detectors=$K4GEO/FCCee/CLD/compact/{DetectorModelList_[0]}/{DetectorModelList_[0]}.xml"+
-            " --inputFiles " + inputFile + " --outputBasename  " + outputFileName+ 
-            f" --VXDDigitiserResUV={ResVDX_UV_[0]}" + 
-            " --trackingOnly" + 
-            " -n " + Nevt_per_job
+            # f" --GeoSvc.detectors=/afs/cern.ch/work/g/gasadows/k4geo/FCCee/CLD/compact/{DetectorModelList_[0]}_3T/{DetectorModelList_[0]}.xml"+
+            f" --GeoSvc.detectors=$K4GEO/FCCee/CLD/compact/{DetectorModelList_[0]}/{DetectorModelList_[0]}.xml"
+            + " --inputFiles "
+            + inputFile
+            + " --outputBasename  "
+            + output_file_name
+            + f" --VXDDigitiserResUV={ResVDX_UV_[0]}"
+            + " --trackingOnly"
+            + " -n "
+            + Nevt_per_job
         )
         command = f"k4run {steering_file} " + arguments + " > /dev/null"
 
         # Write bash script for job execution
         bash_script = (
             "#!/bin/bash \n"
-            f"source {setup} \n"
-            "git clone https://github.com/gaswk/CLDConfig.git \n"
+            f"source {environ_path} \n"
+            "git clone https://github.com/gaswk/CLDConfig.git \n"  # FIXME: new repo location to config (to switch easily to local fork)
```

Comment:
Do you want to keep the cloning in the batch jobs? Talking mostly about ILDConfig now, but these can become quite large and cloning can take quite a bit. If it's at all possible to lift the cloning to a "general setup step" that happens before the submission, etc. happens, I would recommend doing that. The obvious disadvantage is that it becomes user responsibility to make sure that things are consistent and that the config that was used for submission is reproducible.

Potentially something for another PR.